plugins/outputs/cloudwatch/convert_otel.go

```diff
@@ -144,6 +145,42 @@ func ConvertOtelHistogramDataPoints(
 	return datums
 }
 
+// ConvertOtelHistogramDataPoints converts each datapoint in the given slice to
+// Distribution.
+func ConvertOtelExponentialHistogramDataPoints(
+	dataPoints pmetric.ExponentialHistogramDataPointSlice,
+	name string,
+	unit string,
+	_ float64,
+	entity cloudwatch.Entity,
+) []*aggregationDatum { //nolint:revive
+
+	datums := make([]*aggregationDatum, 0, dataPoints.Len())
+	for i := 0; i < dataPoints.Len(); i++ {
+		dp := dataPoints.At(i)
+		attrs := dp.Attributes()
+		storageResolution := checkHighResolution(&attrs)
+		aggregationInterval := getAggregationInterval(&attrs)
+		dimensions := ConvertOtelDimensions(attrs)
+		ad := aggregationDatum{
+			MetricDatum: cloudwatch.MetricDatum{
+				Dimensions:        dimensions,
+				MetricName:        aws.String(name),
+				Unit:              aws.String(unit),
+				Timestamp:         aws.Time(dp.Timestamp().AsTime()),
+				StorageResolution: aws.Int64(storageResolution),
+			},
+			aggregationInterval: aggregationInterval,
+			entity:              entity,
+		}
+		// Assume function pointer is valid.
+		ad.expHistDistribution = exph.NewExpHistogramDistribution()
+		ad.expHistDistribution.ConvertFromOtel(dp, unit)
```

Comment:
nit: If we store the unit in the MetricDatum already, why do we need to store it in the distribution?